docs/contribute-module.md

```diff
@@ -1,81 +1,73 @@
+    ?> *TODO* Update these steps to cover everything for new and updated repos.
+    1.  If you're creating a module, follow these steps:
+        1.  Create repo from the [terraform-ibm-module-template](https://github.com/terraform-ibm-modules/terraform-ibm-module-template).
+    1.  If you're updating a module, follow these steps:
+        1.  Fork the project, clone your fork, and configure the remotes.
+1.  Clone and configure your new repo:
+    1.  Clone into the current directory:
 
-4. Commit your changes in logical chunks. Use Git's [interactive rebase](https://help.github.com/articles/interactive-rebase)
-   feature to tidy up your commits before making them public.
+        ```bash
+        git clone https://github.com/<your-username>/<repo-name>
+        ```
+    1.  Navigate to the newly cloned directory:
 
-5. Locally merge (or rebase) the upstream development branch into your topic branch:
+        ```bash
+        cd <repo-name>
+        ```
+    1.  Assign the original repo to the upstream remote:
 
-   ```bash
-   git pull [--rebase] upstream <dev-branch>
-   ```
+        ```bash
+        git remote add upstream https://github.com/<upstream-owner>/<repo-name>
+        ```
+    1.  If you cloned a while ago, get the latest changes from upstream:
 
-6. Push your topic branch up to your fork:
+        ```bash
+        git checkout <dev_branch>
+        git pull upstream <dev_branch>
+        ```
+    1.  Create a topic branch (off the main project development branch) to contain your feature, change, or fix:
 
-   ```bash
-   git push origin <topic-branch-name>
-   ```
+        ```bash
+        git checkout -b <topic-branch-name>
+        ```
+1.  Commit your changes in logical chunks. Use Git's [interactive rebase](https://help.github.com/articles/interactive-rebase)
+   feature to tidy up your commits before you make them public.
+1.  Locally merge (or rebase) the upstream development branch into your topic branch:
 
-7. Open a pull request with a clear title and description.
-
-8. Make sure the github workflows are successfully running over the PR.
-
-**IMPORTANT**: By submitting a patch, you agree to allow the project owner to
-license your work under the same license as that used by the project.
+    ```bash
+    git pull [--rebase] upstream <dev-branch>
+    ```
+1.  Push your topic branch up to your fork:
 
-NOTE: Be sure to merge the latest from "upstream" before making a pull request!
+    ```bash
+    git push origin <topic-branch-name>
+    ```
+1.  Open a pull request with a clear title and description. Be sure to merge the latest from "upstream" before you create the PR.
+1.  Make sure that the github workflows are successfully running over the PR.
 
-For more info [refer](https://github.com/terraform-ibm-modules/terraform-ibm-issue-tracker/blob/main/README.md).
+!> **Important** By submitting a patch, you agree to allow the project owner to license your work under the same license as that used by the project.
```

Comment:
"By contributing content" instead of "submitting a patch" . It's to make it broader - as the license would apply to anything that is contributed. 'Patch' has got a narrow meaning.

Reply:
Our styles rules caution against using `patch` too. I hadn't touched it because I wasn't sure what legal review had been agreed to. I'll update to content though.